### DISCUSSION TITLE
Filesystem mode - Improvement

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#453] [Ideas]
Sometimes i (and others) find Filesystem browsing more "familiar" (from Windows view i guess).

It will be nice to have two elements:

1) The total number (as an info) of folders or anything else you want
2) The most serious one. When i am at the third page for example, i open a folder and then press "Back" or the "..." to go to the upper level, then i am at the first page and i have to go all that through again. As you imagine this is very frustrating to anyone :)

Thank you JC. 

PS. 6.12.1 looks very good!!!

**Comment by jcorporation** [comment]
1) I do not understand

2) Yes this could be anoying

**Reply by Emilot** [reply]
1) Nevermind.

**Comment by jcorporation** [comment]
I opened an issue for 2: #454
